UnityProject/Assets/Scripts/Player/PlayerSprites.cs

```diff
@@ -188,10 +188,11 @@ public void SubSetBodyPart(BodyPart Body_Part, string path)
 			}
 
 
-			foreach (var bodyPartOrgan in Body_Part.ContainBodyParts)
+			for (int i = 0; i < Body_Part.ContainBodyParts.Count; i++)
```

Comment:
Why does this need to be a `for` loop?

Reply:
It breaks enumeration if it's for each  , because stuff gets added and removed

Reply:
I see. Does that not break the iteration order then (skipping elements or looping one twice)?
Maybe it would make sense to just copy the list and iterate over that?

I don't know the related code, so I can't tell if something could break or if it's completely fine.

Reply:
should be fine, if there's any issues we know what it is

Reply:
should be fine, if there's any issues we know what it is